### DISCUSSION TITLE
Is there a tar of all the icons available?

### DISCUSSION THREAD

**Discussion by Bag72** [homarr-labs/dashboard-icons#589] [Q&A] [answered]
Thanks for all the hard work put in here.  This is a great set of icons for my dashboard.

When I first set up my dashboard I'm sure I downloaded the set to my icons directory.  I must have taken them all down at once because there is no way I would have downloaded them all individually with WGET or CURL.

I want to set up a regular update of the icons and would like to do that via a single file.    Is one available and if so where is it?

**Comment by lammersbjorn** [comment, marked as answer]
Hey, something like this should work `curl https://codeload.github.com/walkxcode/dashboard-icons/tar.gz/main | tar -xz --strip=2 dashboard-icons-main/png`

**Reply by lammersbjorn** [reply]
You can setup a cron job that runs this every so often

**Reply by Bag72** [reply]
Thank you very much.   I appreciate your work on this.

**Reply by Bag72** [reply]
Just though I'd come back and mention.  in Homepage when you add an icon into its home directory when Homepage starts it loads them all into memory so when you add a new link it has to manipulate them all every time not just the ones it needs.

Mine ground to a halt because I put all the icons in there.    So be careful.   It may be the same with other dashboards.
